pkg/users/git.go

```diff
@@ -63,8 +59,7 @@ func (r *GitUserResolver) Resolve(user *gits.GitUser) (*jenkinsv1.User, error) {
 	if r == nil || user == nil {
 		return nil, nil
 	}
-	selectUsers := func(id string, users []jenkinsv1.User) (string, []jenkinsv1.User,
-		*jenkinsv1.User, error) {
+	selectUsers := func(user *gits.GitUser) (*jenkinsv1.User, error) {
```

Comment:
so this is still an intermediate step, right? The aim is still to get rid of passing the function?

Reply:
Done in the WIP commit below

Reply:
I need to refactor some of the tests now, but it kind of works